homeassistant/components/samsungtv/media_player.py

```diff
@@ -187,7 +200,7 @@ def device_class(self):
 
     def turn_off(self):
         """Turn off media player."""
-        self._end_of_power_off = dt_util.utcnow() + timedelta(seconds=15)
+        self._end_of_power_off = dt_util.utcnow() + SCAN_INTERVAL_PLUS_OFF_TIME
```

Comment:
Do we need to reset `self._end_of_power_off` to `None` somewhere?

Reply:
We could set it to `None` in `def update` `if self._end_of_power_off is not None and not self._power_off_in_progress():` set to `None`, but it doesn't look like it will change the functionality besides a small micro-optimization of not having the calculate the time difference since `self._power_off_in_progress()` will already return `False` once the time is elapsed.